### DISCUSSION TITLE
Link to relationship data entry in table view

### DISCUSSION THREAD

**Discussion by SamuelBlickle** [Budibase/budibase#5089] [Help] [answered]
Hi,

in my Budibase DB I have created two data source tables. One holds information about products and the other about suppliers. Every product has one supplier, which gets linked via a 1:n relationship.

When I let BB autogenerate screens for both tables, the list screen of the products shows a table, including a column for the supplier. The names of the suppliers shown in this column even have a gray background.

I am searching for a way to enable the following behaviour: When I click on the gray backgrounded name of a supplier in the products table i want to

- get directed to the detail screen of this supplier *or*
- get redirected to the suppliers list screen, filtered to only show this supplier

I have noticed that this behaviour is implemented in the backend data view of my application. Clicking on the supplier of a product there leads me to the suppliers data as needed. However this doesn't work in the user-facing screens of my app.

Is there something I can do to achieve this behaviour?
Thank you very much for any help!

Best regards,
Samuel Blickle

**Comment by mjashanks** [comment, marked as answer]
HI Samuel.

There's no way to create this behaviour exactly - i.e. click the grey background to view.

However, you can achieve the same result by adding a "View Supplier" button into the table.

1. Click on your table
2. Add a button component
3. Configure its `Define Actions` to `Navigate To` `/suppliers/{{ Table.Products.Supplier._id }}` - you'll need to change your bindings, but should look something like that

**Comment by SamuelBlickle** [comment]
Thank you very much @mjashanks, this does the trick!
However it is important to remark that you need to disable the checkbox 'Link table rows' for this to work. Took me a while to figure that out.
